web-components/src/components/icons/Spinner.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
```

Comment:
will this replace the <Loading /> component I made in 825? Or should that component use this icon? https://github.com/regen-network/regen-web/blob/800bbeeae135068561deb1e8c89037bc404aacb9/web-components/src/components/loading/Loading.tsx

Reply:
probably something to ask to @erikalogie whether we want to use the same spinner or not